recipes/rapidjson/all/conanfile.py

```diff
@@ -1,32 +1,34 @@
-from conans import ConanFile, tools
+from conan import ConanFile
+from conan.tools.files import get, copy
 import os
 
-required_conan_version = ">=1.33.0"
+required_conan_version = ">=1.46.0"
 
 
 class RapidjsonConan(ConanFile):
     name = "rapidjson"
     description = "A fast JSON parser/generator for C++ with both SAX/DOM style API"
-    topics = ("conan", "rapidjson", "json", "parser", "generator")
+    topics = ("rapidjson", "json", "parser", "generator")
     url = "https://github.com/conan-io/conan-center-index"
     homepage = "http://rapidjson.org"
     license = "MIT"
     no_copy_source = True
 
-    @property
-    def _source_subfolder(self):
-        return "source_subfolder"
-
     def source(self):
-       tools.get(**self.conan_data["sources"][self.version], strip_root=True, destination=self._source_subfolder)
+       get(self, **self.conan_data["sources"][self.version], strip_root=True,
+                    destination=self.source_folder)
 
     def package(self):
-        self.copy(pattern="license.txt", dst="licenses", src=self._source_subfolder)
-        self.copy(pattern="*", dst="include", src=os.path.join(self._source_subfolder, "include"))
+        copy(self, pattern="license.txt", src=self.source_folder, dst=os.path.join(self.package_folder, "licenses"))
+        copy(self, pattern="*", src=os.path.join(self.source_folder, "include"), dst=os.path.join(self.package_folder, "include"))
 
     def package_id(self):
-        self.info.header_only()
+        self.info.clear()
 
     def package_info(self):
+        self.cpp_info.set_property("cmake_file_name", "RapidJSON")
+        self.cpp_info.set_property("cmake_target_name", "RapidJSON::RapidJSON")
```

Comment:
```suggestion
        self.cpp_info.set_property("cmake_target_name", "rapidjson")
```

This target name was wrong in cmake_find_package[_multi]: https://github.com/Tencent/rapidjson/blob/06d58b9e848c650114556a23294d0b6440078c61/RapidJSONConfig.cmake.in

Reply:
🤔 https://github.com/Tencent/rapidjson/blob/v1.1.0/RapidJSONConfig.cmake.in

Cool let's follow latest